src/BaselineOfVersionner/BaselineOfVersionner.class.st

```diff
@@ -0,0 +1,47 @@
+Class {
+	#name : #BaselineOfVersionner,
+	#superclass : #BaselineOf,
+	#category : #BaselineOfVersionner
+}
+
+{ #category : #baseline }
+BaselineOfVersionner >> baseline: spec [
+	<baseline>
+	| repository |
+	repository := self packageRepositoryURL.
+
+	spec for: #common do: [
+		
+		spec baseline: 'SUnit' with: [spec repository: repository]..
+		
+		spec
+			package: 'Versionner-Core-Model';
+			package: 'Versionner-Core-Commands';
+			package: 'Versionner-Core-Announcements';
+			package: 'Versionner-Core-DependenciesModel';
+			package: 'Versionner-Spec-Browser';
```

Comment:
Shouldn't this one depend on Spec baseline created in the same PR?

Reply:
Probably. I just extracted the baselines as they were. They had no dependency information before either.